src/app/station/[stationCrs]/page.tsx

```diff
@@ -1,27 +1,38 @@
 import { fetchFromAPI } from "@/app/apiFetch";
-import { getDepartureTable } from "./departureTable"
+import { getDepartureTable } from "./departureTable";
 
 type StationDetailsResponse = {
     location: {addressLines: string, postCode: string},
-    facilities: {fares: {ticketOffice: {openingTimes: string}}}
+    facilities: {fares?: {ticketOffice?: {openingTimes?: string}}}
 }
-async function getStationDetails(stationCrs: string) {
-    const data = await fetchFromAPI<StationDetailsResponse>(`stationDetails/${stationCrs}`)
+
+type StationDescription = {
+    address: string,
+    ticketOfficeInfo: string
+};
+
+async function getStationDetails(stationCrs: string): Promise<StationDescription> {
+    const data = await fetchFromAPI<StationDetailsResponse>(`stationDetails/${stationCrs}`);
     const address = data.location.addressLines.replaceAll("<br>", ", ") + ", " + data.location.postCode;
-    const ticketOfficeOpeningTimes = data.facilities.fares.ticketOffice.openingTimes.replaceAll("<br>", ", ");
+    const ticketOfficeDetailsAvailable = data.facilities.fares 
+        && data.facilities.fares.ticketOffice
+        && data.facilities.fares.ticketOffice.openingTimes;
+
+    const ticketOfficeInfo = ticketOfficeDetailsAvailable 
+        ? "Ticket office opening times: " + data.facilities.fares!.ticketOffice!.openingTimes!.replaceAll("<br>", ", ")
+        : "Ticket office information is unavailable";
```

Comment:
```suggestion
    const ticketOfficeOpeningTimes : string | undefined = data.facilities.fares?.ticketOffice?.openingTimes;

    const ticketOfficeInfo = ticketOfficeOpeningTimes
        ? "Ticket office opening times: " + ticketOfficeOpeningTimes.replaceAll("<br>", ", ")
        : "Ticket office information is unavailable";
```

Let me know if it's not clear why that works.